.editorconfig

```diff
@@ -0,0 +1,6 @@
+[*]
```

Comment:
Suggestion: Should this be validated in CI?

Issue: If editorconfig is going to be expected, it should be called out in the README.

Reply:
Maybe? Adding checking in CI is quite a bit more work. The main thing I wanted to sort here was the newlines & autocrlf -- the other bits were things which kinda seemed like a sensible thing to add at the same time.

Happy to add EditorConfig to the README if you think that's useful.

Reply:
https://github.com/srobo/team-emails/pull/194/commits/39306a6897f4f4c0b05cd6db4df7741757dfa568